app_feup/lib/controller/networking/network_router.dart

```diff
@@ -225,6 +226,78 @@ class NetworkRouter {
     return tripList;
   }
 
+  // Gets the next arrivals for a given bus stop
+  // Uses FlutterWebviewPlugin in order to scrap the information after the
+  // Javascript is loaded
+  static Future<List<Trip>> getNexArrivalsAfterLoadingJS(
```

Comment:
This is getting way too big, can we refactor bus stops functions into its own file under networking?